src/systems/cameras.js

```diff
@@ -0,0 +1,28 @@
+// Used for tracking and managing camera tools in the scene
+AFRAME.registerSystem("cameras", {
+  init() {
+    this.cameraEls = [];
+  },
+
+  register(el) {
+    this.cameraEls.push(el);
+    el.addEventListener("ownership-changed", this._onOwnershipChange);
+    this.currentCamera = null;
+  },
+
+  deregister(el) {
+    this.cameraEls = this.cameraEls.filter(c => c !== el);
+    el.removeEventListener("ownership-changed", this._onOwnershipChange);
+    this.currentCamera = null;
+  },
+
+  getCurrent() {
```

Comment:
What does it mean for a camera to be "current", maybe this should be `getMyCamera"